lib/addStyles.js

```diff
@@ -209,19 +219,19 @@ function addStyle (obj, options) {
 
 	// If a transform function was defined, run it on the css
 	if (options.transform && obj.css) {
-	    result = options.transform(obj.css);
-
-	    if (result) {
-	    	// If transform returns a value, use that instead of the original css.
-	    	// This allows running runtime transformations on the css.
-	    	obj.css = result;
-	    } else {
-	    	// If the transform function returns a falsy value, don't add this css.
-	    	// This allows conditional loading of css
-	    	return function() {
-	    		// noop
-	    	};
-	    }
+			result = options.transform(obj.css);
```

Comment:
I though it is github display madness with tabs tbh (I will send a style PR converting to spaces 😛). 
But in case it's not @tobias-zucali please use the indent the rest of the code currently uses.